README.md

```diff
@@ -2,4 +2,5 @@
 A practice repository. Nothing is safe here.
 
 some changes
+some more changes, trying a pull request
```

Comment:
could you be more specific here about why you're trying a pull request, rather than something more productive you could be doing with your time